doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+FORMAT: 1A
```

Comment:
Taking into account the name of the other .apib file (doc/apiary/v2/fiware-ngsiv2-reference.apib) I'd suggest to use fiware-ngsiv2-cookbook.apib for this one (we would like to keep the word "simple" at the end?).